app/views/stripe_connections/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= link_to(t('stripe_connections.create'), '/auth/stripe_connect', type: 'button') %>
```

Comment:
I don't have a clear idea of where this view is used -- can you clarify or remove?

Blocking, but clarifying is enough to unblock.

Reply:
Leftover from when we had an index on StripeConnections. Deleting.